README.rst

```diff
@@ -271,6 +271,31 @@ Slave node
           name: repuser
           password: password
 
+Client
+------
+
+.. code-block:: yaml
+
+    postgresql:
+      client:
+        server:
+          server01:
+            admin:
+              host: database.host
+              port: 5432
+              user: root
+              password: password
+            mydb:
```

Comment:
Metadata structure is the same as mysql.client + postgresql.server.database done by you